### PR TITLE
add compat data for MediaError

### PR DESCRIPTION
**Pull request by gsouquet**
https://developer.mozilla.org/en-US/docs/Web/API/MediaError

### CODE REVIEW

File: api/MediaError.json [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+            "version_added": true
+          },
+          "edge_mobile": {
+            "version_added": true
+          },
+          "firefox_android": {
+            "version_added": true
+          },
+          "opera_android": {
+            "version_added": true
```
Comment (jpmedley) [inline api/MediaError.json]: Add Opera with value true.

File: api/MediaError.json [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+              "version_added": true
+            },
+            "edge_mobile": {
+              "version_added": true
+            },
+            "firefox_android": {
+              "version_added": true
+            },
+            "opera_android": {
+              "version_added": true
```
Comment (jpmedley) [inline api/MediaError.json]: Add Opera with true

File: api/MediaError.json [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+            "chrome_android": {
+              "version_added": "59"
+            },
+            "firefox_android": {
+              "version_added": "51",
+              "notes": [
+                "While the <code>message</code> property was introduced in Firefox 51, it was only included in nightly builds since it was not part of the specification at that time. As of Firefox 52, it's now available in all builds, including release, since it's now part of the HTML DOM specification."
+              ]
+            }
+          },
```
Comment (jpmedley) [inline api/MediaError.json]: Add opera and opera_android with value 46.

**Comment by gsouquet** [comment]
Just fixed the requested changes, thanks @jpmedley

File: api/MediaError.json [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+      },
+      "message": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/MediaError/message",
+          "support": {
+            "chrome": {
+              "version_added": "59"
+            },
+            "firefox": {
+              "version_added": "51",
```
Comment (Elchi3) [inline api/MediaError.json]: 52

File: api/MediaError.json [hunk 5]
```diff
@@ -0,0 +1,133 @@
...
+              "notes": [
+                "While the <code>message</code> property was introduced in Firefox 51, it was only included in nightly builds since it was not part of the specification at that time. As of Firefox 52, it's now available in all builds, including release, since it's now part of the HTML DOM specification."
```
Comment (Elchi3) [inline api/MediaError.json]: Remove this note.

File: api/MediaError.json [hunk 6]
```diff
@@ -0,0 +1,133 @@
...
+              "version_added": "46"
+            },
+            "webview_android": {
+              "version_added": "59"
+            },
+            "chrome_android": {
+              "version_added": "59"
+            },
+            "firefox_android": {
+              "version_added": "51",
```
Comment (Elchi3) [inline api/MediaError.json]: 52

File: api/MediaError.json [hunk 7]
```diff
@@ -0,0 +1,133 @@
...
+              "notes": [
+                "While the <code>message</code> property was introduced in Firefox 51, it was only included in nightly builds since it was not part of the specification at that time. As of Firefox 52, it's now available in all builds, including release, since it's now part of the HTML DOM specification."
```
Comment (Elchi3) [inline api/MediaError.json]: Remove this note

**Comment by gsouquet** [comment]
Fixed all the changes, thanks @Elchi3 for the review 👍

**Pull request merged**